### PR TITLE
#75 Number Internal and External Repairs

### PR DESCRIPTION
**Pull request by chuckgreenman**
fixes: #75 

The internal and external repairs are now numbered on the web view and the PDF report.  The change itself was in the helper.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -5,6 +5,10 @@ AllCops:
     - 'db/**/*'
     - vendor/bundle/**/*
 
+Metrics/AbcSize:
+  Exclude:
+    - app/helpers/in_house_repair_records_helper.rb
```
Comment (chuckgreenman) [inline .rubocop.yml]: Rubocop flags the helper method generate_in_house_repair_string.  While it's AbcSize meets the requirements for triggering the cop, I don't think it violates the spirit of the cop.

File: app/views/conservation_records/treatment_report_pdf.html.erb [hunk 2]
```diff
@@ -92,15 +92,15 @@ table {
 
   <h2>In-House Repairs</h2>
   <ul>
```
Comment (crowesn) [inline app/views/conservation_records/treatment_report_pdf.html.erb]: In the PDF, I noticed that we're using an unordered list and passing numbered values to it - which appears to result in a list with bullets and numbers:
Screen Shot 2020-03-17 at 8 18 26 AM

Maybe we should pass values without numbers and use `<ol>` tag? This is minor I suppose.

Comment (chuckgreenman) [reply app/views/conservation_records/treatment_report_pdf.html.erb]: I think that's very doable.  I'm not sure why that didn't occur to me the first time around.

**Comment by chuckgreenman** [comment]
@crowesn Changed the bullets to numbers and removed the numbers I was adding in the helper.

**Comment by crowesn** [comment]
@chuckgreenman 
Screen Shot 2020-03-18 at 7 49 33 AM

**Comment by chuckgreenman** [comment]
Alright, sorry about the back and forth on this one.  CI is green and I think everything is being numbered appropriately in the right places now. @crowesn

**Pull request merged**